extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -180,6 +181,7 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
         let _maxAjaxCallsPerView:number = 0;
         let _enableResponseHeaderTracking:boolean = false;
         let _hooks:IInstrumentHook[] = [];
+        let _disabledUrls:any = {};
```

Comment:
Defined as an object for performance, so that it can do an indexed lookup.
Did some testing using [] rather than an object and found IE in IE8 mode performance was seconds but as an object the lookup was ok.